Removing WebIDL defaults for various RTP parameters.

taylor-b opened the pull request:
Fixes #1173.

`setParameters` is only expected to be called with a result of
`getParameters` (enforced by the `transactionId` field). So, these
WebIDL defaults serve no purpose. If a parameter has a default value,
`getParameters` will have already returned it.

taylor-b commented:
Addressed comments. Though I didn't add anything for `channels`; a codec's number of channels will be determined entirely from SDP, so there's no need to describe a default.

stefhak commented:
Looks overall good. However, if `channels` can't be changed, should it not be marked as 'Read-only parameter'?

taylor-b commented:
Added "Read-only parameter" to all codec parameter descriptions, since codecs can only be reordered. Something odd I noticed in the process is that the description of `payloadType` says "This value can be set to control which codec should be used to send a given encoding," and I don't see what this is supposed to mean so I removed it.
